Library/Homebrew/cmd/style.rb

```diff
@@ -54,6 +56,20 @@ def check_style_impl(files, output_type, options = {})
     ]
     args << "--auto-correct" if fix
 
+    if options[:exclude].eql?(:FormulaAuditStrict) && !(options.key?(:except) || options.key?(:only))
```

Comment:
`except` and `only` are just options in code. Should `brew style` accept them as options?